lib/deqm_test_kit/evaluate_measure.rb

```diff
@@ -27,7 +27,8 @@ def measure_evaluation_assert_failure(params, measure_id, expected_status: 400)
     end
 
     measure_options = JSON.parse(File.read('./lib/fixtures/measureRadioButton.json'))
-    measure_id_args = { type: 'radio', optional: false, default: 'measure-EXM130-7.3.000', options: measure_options }
+    measure_id_args = { type: 'radio', optional: false, default: 'ColorectalCancerScreeningsFHIR',
+                        options: measure_options }
```

Comment:
```suggestion
    measure_id_args = { type: 'radio', optional: false, default: 'ColorectalCancerScreeningsFHIR',
                        options: measure_options, title: 'Measure ID' }
```
I like this addition to the other suites!!

Reply:
Since these are technically the measure titles, what do you think of something like:
```suggestion
    measure_id_args = { type: 'radio', optional: false, default: 'ColorectalCancerScreeningsFHIR',
                        options: measure_options, title: 'Measure Title' }
```
?